Bottom: 9d82ce7f8109a3abcc697512477b3617f685c46b
Top:    f11fdde2bbd8d395b1bf965a6e536c672b767a2d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-06-06 12:20:00 +0100

Change scroll-wheel bindings to match more GTK applications










---

diff --git a/src/gpcb-menu.res.in b/src/gpcb-menu.res.in
index 47bee40..3517d73 100644
--- a/src/gpcb-menu.res.in
+++ b/src/gpcb-menu.res.in
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 
   scroll-left = {
     Scroll(left)
